### ISSUE TITLE
todo lo ADJ que -> tan ADJ com / ... (spa->cat)

### ISSUE THREAD

**Issue by jordimas** [apertium/apertium-spa-cat#29]
La investigación no fue todo lo clara que se esperaba -> La recerca no va ser tot **el** clara que s'esperava

Aquí "el" és incorrecte.

**Comment by jaumeortola** [comment]
Si volem fer "todo lo clara que" > "tan clara com", suposo que caldria fer una regla en el fitxer `.t1x`. Això és molt incòmode. M'estimo més fer-ho en LRX, però tampoc és massa net. Com ho veus, @MarcRiera?

**Comment by MarcRiera** [comment]
Això correspon al `T1X`, sí. No és una regla especialment complexa, ho preparo.

**Comment by MarcRiera** [comment]
Resolt a https://github.com/apertium/apertium-spa-cat/commit/3e5ed123baee0801fe49dea83244d032eebeffa2

**Comment by jaumeortola** [comment]
Gràcies, Marc. Això funcionarà bé en un percentatge elevat de casos. Però m'imagino possibles problemes: 
> todo lo nuevo que escribía
> procura que todo lo bueno que hay en tu corazón

> todo lo mejor que han podido

Per això deia que el `T1X` és poc flexible per a segons quines coses, sobretot per a tractar amb el context.

**Comment by MarcRiera** [comment]
Mirem als tests si hi ha gaire canvis a pitjor i pensem si cal revertir-ho. Tots els casos problemàtics que esmentes tenen difícil solució.

Penso que ens hem de plantejar seriosament fer el canvi a `apertium-recursive` (hi ha feina començada a la branca https://github.com/apertium/apertium-spa-cat/tree/recursive-spa-cat). Tot i que el format de les regles és completament diferent i a vegades costa d'entendre, permet aplicar les regles més d'una vegada i fer la traducció per capes. En aquest cas en concret, tindríem tot el context davant i darrere i podríem resoldre la concordança.

**Comment by jaumeortola** [comment]
Sí, mirem què passa en els tests. No crec que calgui revertir-ho. Sí que caldrà a afegir altres solucions per a casos específics, per exemple:
todo lo bueno > totes les coses bones

**Comment by jaumeortola** [comment]
@MarcRiera Com es veu en els tests, hi ha una mica de tot. A vegades depèn del verb que va abans o després. Serà laboriós arreglar la majoria de casos. En LRX es podria anar fent (lo > el/tant/les coses... ; que>que/com...; 'remove' en algun cas...). Segurament no és la manera "canònica" de fer-ho, però mentre no tinguem l'`apertium-recursice` és una possibilitat.
